keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/model/EchoNotification.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.model
+
+import com.netflix.spinnaker.keel.api.NotificationConfig
+import com.netflix.spinnaker.keel.api.NotificationFrequency
+import com.netflix.spinnaker.keel.api.NotificationFrequency.NORMAL
+import com.netflix.spinnaker.keel.api.NotificationFrequency.QUIET
+import com.netflix.spinnaker.keel.api.NotificationFrequency.VERBOSE
+import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_COMPLETE
+import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_FAILED
+import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_STARTING
+
```

Comment:
All these things are in service of giving `echo` the exact map payload that it so desperately wants.